# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/angle/checkout/src/libANGLE/validationES3_autogen.h
# Commit: 13282d7a47a5
# Full Hash: 13282d7a47a57a73be8d060efd9645b4d642f03e
# Author: Miko Mynttinen <mikokm@gmail.com>
# Date: 2019-09-08 09:46:52
# Regressor Bug: 1578576
# File Overlap Count: 4
# Description:
#   Bug 1578576 - Part 2: Update to ANGLE 3865 r=jgilbert
#   
#   Depends on D44561
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D44579
# ==============================================================================

diff -r 3064469c073d -r 13282d7a47a5 gfx/angle/checkout/src/libANGLE/validationES3_autogen.h
--- a/gfx/angle/checkout/src/libANGLE/validationES3_autogen.h	Sat Sep 07 13:36:46 2019 +0000
+++ b/gfx/angle/checkout/src/libANGLE/validationES3_autogen.h	Sat Sep 07 13:38:36 2019 +0000
@@ -53,7 +53,7 @@
 bool ValidateClearBufferuiv(Context *context, GLenum buffer, GLint drawbuffer, const GLuint *value);
 bool ValidateClientWaitSync(Context *context, GLsync sync, GLbitfield flags, GLuint64 timeout);
 bool ValidateCompressedTexImage3D(Context *context,
-                                  TextureType targetPacked,
+                                  TextureTarget targetPacked,
                                   GLint level,
                                   GLenum internalformat,
                                   GLsizei width,
@@ -63,7 +63,7 @@
                                   GLsizei imageSize,
                                   const void *data);
 bool ValidateCompressedTexSubImage3D(Context *context,
-                                     TextureType targetPacked,
+                                     TextureTarget targetPacked,
                                      GLint level,
                                      GLint xoffset,
                                      GLint yoffset,
@@ -81,7 +81,7 @@
                                GLintptr writeOffset,
                                GLsizeiptr size);
 bool ValidateCopyTexSubImage3D(Context *context,
-                               TextureType targetPacked,
+                               TextureTarget targetPacked,
                                GLint level,
                                GLint xoffset,
                                GLint yoffset,
@@ -245,7 +245,7 @@
 bool ValidateSamplerParameteri(Context *context, GLuint sampler, GLenum pname, GLint param);
 bool ValidateSamplerParameteriv(Context *context, GLuint sampler, GLenum pname, const GLint *param);
 bool ValidateTexImage3D(Context *context,
-                        TextureType targetPacked,
+                        TextureTarget targetPacked,
                         GLint level,
                         GLint internalformat,
                         GLsizei width,
@@ -269,7 +269,7 @@
                           GLsizei height,
                           GLsizei depth);
 bool ValidateTexSubImage3D(Context *context,
-                           TextureType targetPacked,
+                           TextureTarget targetPacked,
                            GLint level,
                            GLint xoffset,
                            GLint yoffset,